rclpy/rclpy/executors.py

```diff
@@ -731,19 +732,28 @@ class MultiThreadedExecutor(Executor):
     """
     Runs callbacks in a pool of threads.
 
-    :param num_threads: number of worker threads in the pool. If ``None``, the number of threads
-        will use :func:`multiprocessing.cpu_count`. If that's not implemented the number of threads
-        defaults to 1.
+    :param num_threads: number of worker threads in the pool.
+        If ``None``, the number of threads will be automatically set by querying the underlying OS
+        for the CPU affinity of the process space. If it is not set, defaults to 2.
```

Comment:
Sorry, I think we should make this a bit clearer:

```suggestion
        If ``None``, the number of threads will be automatically set by querying the underlying OS
        for the CPU affinity of the process space. If the OS doesn't provide this information, defaults to 2.
```